tests/qos/qos_sai_base.py

```diff
@@ -934,24 +954,43 @@ def dutConfig(
             src_asic = get_src_dst_asic_and_duts['src_asic']
             dst_dut_index = get_src_dst_asic_and_duts['dst_dut_index']
             dst_asic = get_src_dst_asic_and_duts['dst_asic']
+            src_system_port = src_dut.config_facts(host=src_dut.hostname, source='running')[
```

Comment:
Can we add vendor/chip specific check as it system_port may not be applicable for all ?

Reply:
done